kt/godot-library/src/main/kotlin/godot/core/StringName.kt

```diff
@@ -1,5 +1,6 @@
 package godot.core
 
+import godot.core.memory.GarbageCollector
```

Comment:
Still needed ?

Reply:
All native core times have to register themselves to the Garbage Collector.
I moved the garbage collector to a new package so the import is necessary now.